jumpstart_ui.info.yml

```diff
@@ -2,15 +2,15 @@ name: 'Jumpstart UI'
 type: module
 description: 'Provides UI Tools for Jumpstart Projects'
 package: Stanford
-version: 8.x-1.6
+version: 8.x-1.7
 core_version_requirement: '^8.8 || ^9'
 dependencies:
-  - components:components
-  - drupal:block
-  - drupal:breakpoint
-  - ds:ds
-  - ui_patterns:ui_patterns
-  - ui_patterns_layouts:ui_patterns_layouts
-  - ui_patterns_library:ui_patterns_library
+    - 'components:components'
+    - 'drupal:block'
+    - 'drupal:breakpoint'
+    - 'ds:ds'
+    - 'ui_patterns:ui_patterns'
+    - 'ui_patterns_layouts:ui_patterns_layouts'
+    - 'ui_patterns_library:ui_patterns_library'
```

Comment:
We have to change the script that creates these PRs. this is incorrect and requires manual changes still. for now i will make the change, but we need to fix the script for the next run

Reply:
acknowledged.